### PR TITLE
Adds foreign currency expenses process

### PR DESCRIPTION
**Pull request by rub1e**
Answers #95 - documents foreign expenses process

### CODE REVIEW

File: finances.md [hunk 1]
```diff
@@ -79,6 +79,34 @@ Bookkeeping is the activity or occupation of keeping records of the financial af
 The expenses policy can be found here:
 https://docs.google.com/document/d/1SeXnzTD4P-Ley-RfcEpM4X5aYgjB8SN9mwywtXMPi-o/edit. Expenses are captured in Xero according to the policy document.
```
Comment (iteles) [inline finances.md]: I know this isn't part of this PR (It appears to have been pushed straight to master many months ago), but having an _edit_ link here isn't very clever. Can we (not a priority) change it to a view link?

File: finances.md [hunk 2]
```diff
@@ -79,6 +79,34 @@ Bookkeeping is the activity or occupation of keeping records of the financial af
...
 
+## Foreign currency expenses
+
+The reasons for this ludicrous process are explained [here](https://github.com/dwyl/hq/issues/430#issuecomment-376168913).
```
Comment (iteles) [inline finances.md]: 👍

File: finances.md (hunk 1 above)
Comment (rub1e) [reply finances.md]: @iteles Weirdly I only have view access - I couldn't tell where the thing was saved so I've added it to the `0. Accounts` folder and I'll change the link when I tackle the whole folder structure

Comment (iteles) [reply finances.md]: @rub1e Looks like it's in the public dwyl finances folder:
https://drive.google.com/drive/folders/0B0XeQ2t5WSK2QVdWaGpMekw2WkU

Comment (rub1e) [reply finances.md]: @iteles iiiiiiiinteresting 🤔 

How would one gain access to this supposedly public folder? Asking for a friend

(I can't access the folder and I still only have view access to the file)

Comment (iteles) [reply finances.md]: I guess I could share it with you? I assumed the whole folder was supposed to be available to folks. Set up by our friendly neighbourhood 👻 so we can just figure it out, no rush 👍

**Pull request merged**